src/app/components/dynamic-list/DynamicList.test.jsx

```diff
@@ -34,7 +34,7 @@ describe("DynamicList", () => {
         };
         const component = mount(<DynamicList {...dynamicListProps} />);
         it("should display a dynamic list", () => {
-            expect(component.find(".dynamic-list__title").length).toBe(1);
+            expect(component.find('[data-testid="dynamic-list-title"]').exists()).toBe(true);
```

Comment:
the `data-testid` is the api for react testing library which you are not using here, you use enzyme in this file. 
the way you use it in test is :
`ByTestId('dynamic-list-title') ` for example `const element = screen.getByTestId('dynamic-list-titlet')`
sorry Tim confused you here I think, you should revert to classname :-)